terraform/modules/github_repository/main.tf

```diff
@@ -78,25 +78,9 @@ resource "github_repository_collaborators" "admins" {
     permission = "admin"
     team_id    = "service-accounts" # id: 6206668
   }
-  user {
-    permission = "admin"
-    username   = "cloudeteerbot"
-  }
```

Comment:
cloudeteerbot cannot be deleted here.

in a deployment situation it can be a race condition (on new repositories) where the process first removes cloudeteerbot and then tries to add team "service-accounts".

This way the cloudeteerbot locks himself out.